main.go

```diff
@@ -87,18 +89,37 @@ func run(coverprofile string, args []string, covermode, cpu, parallel, timeout s
 	defer file.Close()
 	// pkgs is packages to run tests and get coverage.
 	var pkgs []string
+	var coverpkgs []string
 	for _, pkg := range args {
 		ps, err := getPkgs(pkg)
 		if err != nil {
 			return err
 		}
 		pkgs = append(pkgs, ps...)
+
 	}
 	if len(pkgs) == 0 {
 		pkgs = []string{"."}
+
+	}
+
+	for _, pkg := range pkgs {
+		var cmd *exec.Cmd
+
+		if tags != "" {
+			cmd = exec.Command("go", "build", "-tags", tags, pkg)
+		} else {
+			cmd = exec.Command("go", "build", pkg)
```

Comment:
You are right, I just updated it to send the build output to `/dev/null`. Sadly `go test` is not enough though, because it can be that there is a test file in the directory so that won't fail. I also removed the check if no `-tags` argument is supplied as it is not really useful then.